### DISCUSSION TITLE
[Feature] Dedicated search/filter function to filter album names and descriptions.

### DISCUSSION THREAD

**Discussion by gguglietti** [immich-app/immich#2854] [General]
### The feature

A search function to find/filter album by names and description. When having multiple albums, additional search / sorting functionality beyond alphabetically / creation date would be beneficial. For example, the user would be able to search albums for particular types of events (i.e. "trips") by searching for the album name or description keyword. The album page would then show only albums which contain that keyword. Alternatively, the search functionality could be added to the search page, being able to display albums as results, in addition to the existing individual photo/video search. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mrupnikm** [comment]
This is a much needed feature for me as well. Atleast to put the albums to the main search

**Reply by alextran1502** [reply]
Done in #2900 available in the next release
